python/feathub/feature_tables/tests/test_redis_source_sink.py

```diff
@@ -404,6 +404,65 @@ def _test_redis_sink_custom_key(
     redis_client.close()
 
 
+def _test_redis_sink_not_keep_timestamp(
+    self: FeathubITTestBase,
+    host: str,
+    port: int,
+    mode: str,
+    redis_client: Union[Redis, RedisCluster],
+):
+    # TODO: Fix the bug that in flink processor when column "val"
```

Comment:
Can you create a JIRA and provide JIRA in this comment?

Reply:
It seems that this problem no longer exist in Feathub after previous code updates and Flink version upgrade. I have removed all such TODOs and add test case that verify output results when the input contains None values.